machines/vm/my-service.example.com.yaml

```diff
@@ -0,0 +1,33 @@
+# virtual machines deployed to
```

Comment:
I'm reluctant to build in a split between VM and non VM, particularly because it should be possible to handle this with some combination of templates and `params` values in the config and maybe a few small additions/adjustments to existing structs.

Since we're really only talking about a difference in config properties that will be used in templating later, I could maybe see a "plug-in" approach where we maintain different definitions of the Machine struct that only have to adhere to a basic contract that requires the current minimal set of properties (some solution with composition using a BaseMachine struct should do it).  From there, those who need it could then use build tags if they need a more complex `Machine` structure.